src/app/components/favorite-libraries/favorite-libraries.component.ts

```diff
@@ -86,6 +86,10 @@ export class FavoriteLibrariesComponent extends PageComponent implements OnInit
     }
   }
 
+  handleNodeClick(event: Event) {
```

Comment:
is this required?

Reply:
Unfortunately yes, it's the way to revolve the typing issues that Ivy raises for DOM events